django_cloud_tasks/tasks.py

```diff
@@ -261,3 +262,42 @@ def __client(self):
         )
 
         return CloudPublisher(publisher_options=publisher_options)
+
+
+class RoutineTask(Task):
+    def run(self, routine_id: int):
+        routine = models.Routine.objects.get(pk=routine_id)
+        if routine.status == "completed":
+            self._run_next_routines(routine=routine)
+            return
+
+        if self._exists_pending_routine(routine=routine):
+            routine.enqueue()
```

Comment:
Since the previous routine triggers its next once it's finished, it's not necessary to re-queue.
Right?

Reply:
There is no validation about that. I don't think we must assure as well. By following diagram the previous is a dependency to run a routine

<img width="608" alt="Screen Shot 2022-02-14 at 8 28 53 PM" src="https://user-images.githubusercontent.com/5209129/153964259-9bb2205a-7e29-41ff-904d-96d834319968.png">
e